stl/inc/format

```diff
@@ -1776,7 +1776,7 @@ struct _Format_arg_traits {
         -> basic_string_view<_Char_type>; // not defined
 
     template <class _Traits, class _Alloc>
-    static auto _Phony_basic_format_arg_constructor(const basic_string<_Char_type, _Traits, _Alloc>&)
+    static auto _Phony_basic_format_arg_constructor(basic_string<_Char_type, _Traits, _Alloc>)

```

Comment:
2. Is there a related issue with the `const _Char_type*` overload:
   https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/stl/inc/format#L1772
   My brain-compiler suggests that `_Char_type*` will go through the general `handle` codepath.

Reply:
This sounds like the `char`-type flavor of LWG-3540.